test/e2e/e2e-test/docker/kafka/docker-compose.log.yml

```diff
@@ -31,33 +31,34 @@ services:
         condition: service_healthy
       broker-b:
         condition: service_healthy
-  provider_kafka:
+
+  sw-satellite:
     build:
       context: ../../../
-      dockerfile: e2e-test/docker/kafka/Dockerfile.log.producer
-      args:
-        - SW_AGENT_JDK_VERSION=${SW_AGENT_JDK_VERSION}
+      dockerfile: e2e-test/docker/Dockerfile.satellite
     networks:
       - e2e
     expose:
-      - 8080
+      - 11800
     volumes:
-      - ../../../../jacoco:/jacoco
-    environment:
-      JAVA_OPTS: >-
-        -javaagent:/jacoco/jacocoagent.jar=classdumpdir=/jacoco/classes/provider,destfile=/jacoco/provider.exec,includes=org.apache.skywalking.*,excludes=org.apache.skywalking.apm.dependencies.*
-         -Dbootstrap.service=broker-a:9092,broker-b:9092
-    healthcheck:
-      test: ["CMD", "sh", "-c", "nc -nz 127.0.0.1 8080"]
-      interval: 5s
-      timeout: 60s
-      retries: 120
+      - ./satellite_config.yaml:/configs/satellite_config.yaml
+    command: ["start"]
```

Comment:
Forget to modify this? Should be `/sw-satellite` or remove this line?

Reply:
![image](https://user-images.githubusercontent.com/26432832/106078043-ac897e00-614d-11eb-8074-f0282ea72923.png)

The full command is `/sw-satellite start`, `/sw-satellite` in `ENTRYPOINT`.